pkg/cmd/roachtest/cluster.go

```diff
@@ -3053,6 +3059,7 @@ func (c *clusterImpl) AddInternalGrafanaAnnotation(
 	if err != nil {
 		return err
 	}
+	req.Tags = append(req.Tags, "roachtest")
```

Comment:
Remind me why the tags are different between `AddGrafanaAnnotation` and `AddInternalGrafanaAnnotation`.